content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,525 @@
+weight: 
+description: >
+ CHT example application that uses a third party app to capture the result of a Rapid Diagnostic Test.
+relatedContent: >
+  apps/reference/forms/app/#android-app-launcher
+  apps/reference/forms/app/#cht-xform-widgets
+  apps/reference/forms/app/#parse-timestamp-to-date
+---
+
+Medic has worked with [FIND](https://www.finddx.org) to build a CHT reference application for COVID-19 point-of-care testing with Rapid Diagnostic Tests (RDT). Using the reference app as an example, CHT app developers can easily include the provisioning and capture of RDT in workflows. These workflows can include third-party applications, like [Dimagi's RD-Toolkit](https://github.com/dimagi/rd-toolkit/), that guide health workers through the use of the RDT.
+
+You can find the code for this application in the [CHT Core repository on GitHub](https://github.com/medic/cht-core/tree/master/config/covid-19).
+
+## Problem Being Addressed
+
+The original call for proposals best describes why Medic created this app: 
+
+> FIND is working towards supporting countries in implementing an effective test-trace-isolate response using digital tools. To this end, FIND is looking to partner with leading digital solution providers to accelerate the development and deployment of a set of minimum functionalities for the collection of COVID RDT data and supporting incorporation of these functionalities into existing digital tools for use in low- and middle-income country settings.
+
+## Solution Overview
+
+This reference app provides a base layer of functionality that you can easily customize to meet the needs of your health program.  Specifically, this application has examples of:
+
+ * Provisioning COVID-19 RDT tests 
+ * Capturing COVID-19 RDT results
+ * Storing all RDT data, including pictures, in the CHT
+ * Best practices for RDT workflows in the CHT
+ 
+Your instance of CHT needs to be on v3.13+ which has features developed to support this reference application, including:
+ 
+  * **Display Base64 Image** - show ASCII encoded images inline
+  * **Android App Launcher** - integrate with third party apps like the RD-Toolkit
+  * **Parse Timestamp to Date** -  convert epoch time stamps (`1628945040308`) to your desired format (`Sun Mar 19 13:25:08`).
+  
+For more information on these features, see the ["Related Content"](#related-content).
+
+Additional requirements for this application beyond CHT 3.13, include [CHT Android 0.10.0](https://github.com/medic/cht-android) or later and Dimagi's [RD-Toolkit 0.9.8](https://github.com/dimagi/rd-toolkit/) or later. 
+
+While this application utilizes the RD-Toolkit, the integration features in the CHT Core and CHT Android are generic.  This means you could use a different RDT Android application if you prefer. Beyond the scope of RDTs, you may use any application that is supported.  To read more about this feature, see the [forms reference documentation]({{< ref "apps/reference/forms/app#android-app-launcher" >}}). 
```

Comment:
Helpful new paragraph!

Personal preference to avoid using `utilize`, and have link text match the anchor more closely, along with a clarification:

```suggestion
While this application calls the RD-Toolkit, the integration features in the CHT Core and CHT Android are generic.  This means you could use a different RDT Android application if you prefer. Beyond the scope of RDTs, you could use this integration feature to launch any other Android app to perform an action and save the result in the CHT. To read more about this feature, see the [Android App Launcher section in the Forms reference documentation]({{< ref "apps/reference/forms/app#android-app-launcher" >}}). 
```